src/metabase/async/util.clj

```diff
@@ -35,7 +37,7 @@
       (a/close! out-chan)))
   nil)
 
-(defn cancelable-thread-call
+(defn- cancelable-thread-call
```

Comment:
Only used in this namespace so I made it private. I think a lot of this stuff in this namespace is pretty close to being able to be removed